src/app/portfolio/portfolio.component.html

```diff
@@ -40,26 +40,29 @@
                      src="{{ item.campaign.photo | toUrlIPFS }}" alt="Project image">
               </a>
             </div>
-            <div class="w-full bg-white rounded-b-3xl lg:rounded-none lg:rounded-r-3xl p-6">
-              <div class="flex">
-                <div class="bg-gray-200 px-8 rounded-full py-2 flex items-center">
-                  <span class="font-bold">{{ item.tokenValue | formatUnit:'bignumToWei' | currencyDefault }}</span>
-                  <span class="ml-2">Invested</span>
+            <div class="flex flex-col justify-between w-full bg-white rounded-b-3xl lg:rounded-none lg:rounded-r-3xl
```

Comment:
Funding progress and "Cancel investment" button were not aligned to the bottom of the card.
Before:
![Screenshot 2021-09-22 at 16 10 29](https://user-images.githubusercontent.com/12549625/134380365-17805b75-c64f-4f50-9f71-d1b9648cb73a.png)
After:
![Screenshot 2021-09-22 at 16 17 05](https://user-images.githubusercontent.com/12549625/134380434-a55ea18e-5fbf-4c93-9ece-dc1826f5ae73.png)